handler/transport.go

```diff
@@ -33,9 +34,17 @@ type transport struct {
 }
 
 func (t *transport) RoundTrip(r *http.Request) (*http.Response, error) {
+	for _, urlPath := range t.cfg.BypassURLPaths {
+		if urlPath == r.URL.Path {
+			if err := glg.Info("Authorization checking skipped on: " + r.URL.Path); err != nil {
```

Comment:
thinking we can just ignore the error

Reply:
The bot is still working. Let's keep the checking for consistency?

Reply:
For example 
https://github.com/yahoojapan/authorization-proxy/blob/master/handler/handler.go#L76
it do not check the error...

in my opinion the logging error shouldn't block the user request.

Reply:
It is the old code🙈

https://github.com/yahoojapan/authorization-proxy/pull/20/files
https://github.com/yahoojapan/authorization-proxy/pull/20

the bot is not there when doing the PR

---

update in https://github.com/yahoojapan/authorization-proxy/pull/41/commits/10ef00c659cb742438b86a957345b5e44b55b180, https://github.com/yahoojapan/authorization-proxy/pull/41/commits/526ee3a0371647f13a071547bfed89b541691dd1
use default logger on error

Reply:
just ignore the bot XDD
using default logger is non-sense for me.....
code readability is really bad.... if the logging is failed then the system should be non-functionable

Reply:
1. that log is important for security
2. it does not return error not, the request will still go on

Reply:
as i am not sure where the default logger will write to, so it is meaningless if user/logging system cannot trace the log. 
also the default logger could also fail so it is meaningless for me.

Reply:
the default logger use stderr
https://github.com/golang/go/blob/25d0ffe603885a9c42400b35cb39e93a56a7aa95/src/log/log.go#L76

It will ignore the error internally
https://github.com/golang/go/blob/25d0ffe603885a9c42400b35cb39e93a56a7aa95/src/log/log.go#L326

just as a backup plan, in case glg stdout print fail.
As the log is important, chances of both method to fail should be lower.

Reply:
I don't think we need this XDD 
every logs are important, but basically we can do nothing when the logging is failed....

Reply:
- create another PR to fix the bot config and this line?

- should not crash? as you said, logging should not affect the main function

Reply:
should we just ignore the bot this time?

Reply:
better not, else it is difficult to trace to bot behavior later on

Reply:
I think theres so many places we ignore the error so i dont think it is the concern

Reply:
those errors are ignored, because the bot is added at that time @@

Reply:
i  don't think the bot is the concern, the problem should be wether to add this error handling or not

Reply:
fixed: https://github.com/yahoojapan/authorization-proxy/pull/41/commits/3e09fc0c3eaf2acf0aa15152942a08781e86d7a7

Reply:
issue created: https://github.com/yahoojapan/authorization-proxy/issues/47